### PR TITLE
Add guideline on how to estimate maintenance windows

### PR DESCRIPTION
**Pull request by hcarreras**
A recurring topic. All teams should be able to estimate the maintenance windows in a consistent way.

### CODE REVIEW

File: best-practices/estimating-project-maintenance.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Maintenance window goal
+
+When we stop actively working on a project, the project moves to the maintenance phase. Any project that does not have a full-time team on it is considered to be on maintenance. The maintenance window goal is to ensure that the project does not go to a state where we would not want to work on it. The maintenance window also serves as a pragmatic way of bulking tasks to reduce context-switch.
+The maintenance window is not a tool for moving a project out of the legacy status nor to develop new functionality.
+
+During a planned maintenance window, the DI team will:
+- Update security vulnerabilities (by Dependabot).
+- Check the Asana board for any reported bugs and fix them.
+- Check error tracking (Rollbar) and fix them.
```
Comment (runephilosof-abtion) [inline best-practices/estimating-project-maintenance.md:9]: For easy fixable errors, fix them.
But in other cases, the client should pay outside of the SLA for the fix, so we would normally create a task for it and inform the client.

Comment (hcarreras) [reply best-practices/estimating-project-maintenance.md:9]: SLAs are paid per hour. They pay for 30 hours of scheduled work every X amount of time. 
If a task cannot be completed on the scheduled maintenance window, like a major Rails update, or a tricky bug, we will inform the client.

I'll add this to the description to make sure the process is clear.

**Comment by mathias-abtion** [comment]
I find this to be a quite useful addition to the guidelines. The document seems to mostly focus on Ruby, Rails, and Heroku, though, and I am wondering if we need to cover the .NET stack as well?

**Comment by KlausEnevoldsen-Abtion** [comment]
> I find this to be a quite useful addition to the guidelines. The document seems to mostly focus on Ruby, Rails, and Heroku, though, and I am wondering if we need to cover the .NET stack as well?

Well, it depends. In case of ISS they use Azure DevOps which has it's own implementation of Git. Azure DevOps has some pipelines that can be setup in a similar way.

**Comment by hcarreras** [comment]
Good point. I will clarify that it is the kind of maintenance that we do and that it will vary from app to app depending on the technology

**Comment by hcarreras** [comment]
I've made it language-agnostic :)

**Review by Croesgaard — APPROVED** [review]
Well-informed overview; flow and descriptions makes sense to me. Maybe @SimoneStengaard could have a final look over it to make sure i'm not missing anything from the PMs' side?

**Comment by SimoneStengaard** [comment]
Should there be a small section about if they don't have an SLA / maintenance window? Otherwise super fine :)

**Comment by hcarreras** [comment]
@SimoneStengaard let's take the conversation to a different place to keep this PR to the point. 
We do need to discuss what to do with those clients.

**Pull request merged**